code-studio/src/js/components/progress/course_progress_row.jsx

```diff
@@ -2,29 +2,47 @@
 import React from 'react';
 import { STAGE_TYPE } from './types';
 import StageProgress from './stage_progress.jsx';
+import color from '../../color';
+
+var style = {
```

Comment:
nit: could be const. also our convention has been to call this variable styles rather than style.